frontend/src/pages/start-page/ui/sticker/sticker.component.scss

```
@@ -0,0 +1,65 @@
+.sticker__link {
+  cursor: pointer;
+
+  .sticker {
+
+    &:hover .block__button {
+      max-height: 100px;
+      transition: max-height .8s;
```

В Figma, вроде как. не линейная анимания, поэтому считаю нужным добавить `ease-out`
Это делает анимацию приятнее и плавнее, как по мне 

```suggestion
      transition: max-height .8s ease-out;
```

---

Не нашёл как посмотреть в фигме какая работает анимация, поэтому оставил как "временное решение"((